README.md

```diff
@@ -120,7 +120,7 @@ Check the [systemd-journal plugin of Netdata](https://github.com/netdata/netdata
   <br />
   Netdata actively supports and is a member of the Cloud Native Computing Foundation (CNCF)<br />
   &nbsp;<br/>
-  ...and due to your love :heart: and :star:, <b>Netdata is leading the Observability category</b> at the <a href="https://landscape.cncf.io/card-mode?category=observability-and-analysis&grouping=no&sort=stars">CNCF landscape</a>!
+  ...and due to your love :heart: and :star:, <b>Netdata is leading the Observability category</b> at the <a href="https://landscape.cncf.io/?item=observability-and-analysis--monitoring--netdata">CNCF landscape</a>!
```

Comment:
It looks like they remove sort by GH start, at least I couldn't find how to do that - if so, I think we need to remove the "is leading the Observability category" part.

Reply:
yes, they removed it.
I discussed that with @Aliki92 and we are to keep the text as it is on the website, didn't actually notice it was different here. let me update that

Reply:
updated

Reply:
Thanks!